### DISCUSSION TITLE
Welcome to pybullet_ros Discussions!

### DISCUSSION THREAD

**Discussion by oscar-lima** [oscar-lima/pybullet_ros#24] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by robogeekcanada** [comment]
Thank you for developing this repo. It works well. I had no issues with installation and the instructions are really clear and everything worked well even in WSL.

I will be experimenting more in the upcoming weeks and give my impressions.
